compiler/testData/codegen/bytecodeText/companion/kt14258.kt

```diff
@@ -0,0 +1,19 @@
+// Checks that methods 'access$getMy$p', 'access$getMy$cp' and 'getMy' are not generated and
```

Comment:
Why have you deleted the `kt-14258.kt` test from your earlier commits where a lot more cases were covered? If you think that test was getting too big to understand, you can always split it into several independent tests, e.g. `kt14258_1.kt`, `kt14258_2.kt`, ...

Reply:
Yes, the test was too complicated as your recommend I split it into several.